client/src/config/donorHompageConfig.json

```diff
@@ -0,0 +1,87 @@
+{
+  "map": {
+    "points": [
+      {
+        "x": 0.28,
```

Comment:
Marker with (x = 0.28, y=0.62) will be at (x = 28% map width from map left, y=62% map height from map top)

Reply:
Maybe set it to x = 28 and y = 62 to avoid having to multiply by 100 later.

Reply:
using decimals makes it obvious that we're using a scale of 0 - 1